web/locales/it.json

```diff
@@ -1286,23 +1286,23 @@
             "zoneName": "El Salvador"
         },
         "SX": {
-            "countryName": "Sint Maarten",
-            "zoneName": "Olandese"
+            "countryName": "San Martino",
+            "zoneName": "Neerlandese"
```

Comment:
Here I would say you require the noun of the country not the adjective: `Paesi Bassi`

Reply:
I did it because it's with the adjective in the original string in both the French and the Dutch part

Reply:
that sounds odd though.. there might be a mistake also in the other languages 🤷 .

In that case I am fine with `Neerlandese` 👍